azure_pipelines/scripts/get_automation_conf_file.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import base64
+import os
+import json
+
+def run_az_command(command):
+    try:
+        result = subprocess.run(command, shell=True, capture_output=True, text=True)
+        if result.returncode == 0:
+            return result.stdout.strip()
+        else:
+            print("Error:", result.stderr.strip())
+            return None
+    except Exception as e:
+        print("Exception:", str(e))
+        return None
+# 2 Loading conf.json file as base64 encoding form lab keyvault
+print("Loading conf.json file...")
+output_1 = ""
+output_2 = ""
+# 2.1 Loading the 1st half and formatting 
+az_command_1 = "az keyvault secret show --name \"appleautomationconfjsonfile1\" --vault-name \"buildautomation\" --query \"value\""
+output_1_without_formatting = run_az_command(az_command_1)

```

Comment:
Should we check that the result of the "run_az_command" function isn't "None"?

Reply:
It should be fine to crash the pipeline (sorry not a python export and please correct) and fail the job if anything wrong with the checked-out step. We can always optimize if the current script does not work out. The main purpose of this change is for adopting the SNI cert for next steps